client/packages/invoices/src/Prescriptions/DetailView/Footer/StatusChangeButton.tsx

```diff
@@ -94,7 +99,9 @@ const useStatusChangeButton = () => {
 
   const showPaymentWindow =
     lines != null &&
-    lines.nodes.filter(({ totalAfterTax }) => totalAfterTax > 0).length > 0;
+    lines.nodes.filter(({ totalAfterTax }) => totalAfterTax > 0).length > 0 &&
+    insuranceData?.nodes != null &&
```

Comment:
I want this to be if there are any `InsuranceProviders` in the system at all, not based on if the particular patient has insurance.
The individual patient might not have insurance, but they'll still need to pay.

Also can you add a comment that this should be a global pref, and github issue link here: https://github.com/msupply-foundation/open-msupply/issues/6179

Reply:
Ahh if that's the case then it'll have to wait for Patient Insurance Editing to go in. it has the insurance providers call